ios/Gekidou/Sources/Gekidou/Networking/PushNotification.swift

```diff
@@ -114,24 +114,32 @@ extension Network {
             let group = DispatchGroup()
             
             let channelId = notification.userInfo["channel_id"] as! String
+            let rootId = notification.userInfo.index(forKey: "root_id") != nil ? notification.userInfo["root_id"] as! String : ""
```

Comment:
what about?
```suggestion
            let rootId = notification.userInfo.index(forKey: "root_id") ?? ""
```

Reply:
on the other hand you are already checking for nullability in the `fetchPostsForChannel` function